packages/angular-cli/ember-cli/lib/utilities/windows-admin.js

```diff
@@ -21,8 +21,7 @@ module.exports = {
 
           if (!elevated && ui && ui.writeLine) {
             ui.writeLine(chalk.yellow('\nRunning without elevated rights. ' +
```

Comment:
Actually I _think_ this is false now, since we don't use Broccoli. But I don't have a windows computer to test this. Could you try running it in non-admin, and admin, and say run a `ng build`/`ng version`/`ng test` and see if that makes a difference? If so then we should keep it, otherwise just remove the warning. It's noise :)